src/openforms/utils/tests/test_decorators.py

```diff
@@ -0,0 +1,80 @@
+from django import forms
+from django.contrib.messages import get_messages
+from django.contrib.messages.storage.fallback import FallbackStorage
+from django.db import models
+from django.test import TestCase
+from django.test.client import RequestFactory
+from django.utils.translation import gettext_lazy as _
+
+import requests
+import zds_client
+
+from openforms.utils.decorators import dbfields_exception_handler
+
+
+class TestDbFieldsExceptionHandler(TestCase):
```

Comment:
```suggestion
class DbFieldsExceptionHandlerTests(TestCase):
```
consistent with the rest of the codebase.